azure-pipelines.yml

```diff
@@ -155,23 +178,41 @@ jobs:
     projplatforms: $[ dependencies.generator.outputs['mtrx.projplatforms'] ]
   steps:
   - script: |
-      # If we're on master and this is not a pull request, then DEPLOY.  NOTE: A
-      # manual rebuild of a PR in Azure web interface is not a `PullRequest`
-      DEPLOY=""
-      if [[ "$(Build.Reason)" != "PullRequest" ]] && [[ "$(Build.SourceBranch)" == "refs/heads/master" ]] ; then
-          DEPLOY="--deploy-bin"
-      fi
-
       # Cleanup temporary things that might have been left-over
       ./clean_builds.sh
       ./clean_products.sh
 
       cd $(PROJECT)
-      $(JULIA) ./build_tarballs.jl --verbose ${DEPLOY} $(PLATFORM)
+      $(JULIA) ./build_tarballs.jl --verbose $(PLATFORM)
+
+      # After building, we take the single tarball produced with the proper NAME, and upload it:
+      TARBALLS=( ./products/${NAME}*${PLATFORM}*.tar.gz )
```

Comment:
We should _never_ use `NAME`, this leads to issues similar to #485

Reply:
Heh, yeah, notice that I changed that to `./products/${NAME%@*}*${PLATFORM}*.tar.gz` immediately afterward.  :P

Reply:
Yes, that might actually cover most cases... Let's see how it goes